src/initKonnectors.json

```diff
@@ -56,6 +56,17 @@
   },
   {
     "accounts":[],
+    "slug":"orange_mobile",
```

Comment:
I would prefer the slug `orangemobile` here to be coherent with other connectors but it's what is declared in the `manifest.konnector` of this connector.